client/src/components/UserRegisterForm.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Grid } from "semantic-ui-react";
+
+export class UserRegisterForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      name: "",
+      password: "",
+      confirmationPassword: "",
+      userSignUp: false
+    };
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = e => {
+    e.preventDefault();
+    const { password, confirmationPassword } = this.state;
+    if (password !== confirmationPassword) {
+      alert("passwords dont match");
+    } else {
+      const UserRequest = {
```

Comment:
JS convention is to start the name of a variable with a lower case letter: "userRequest"

Reply:
ok changed